src/test/java/seleautomation/com/HalaTraining1/GitHubtest.java

```diff
@@ -0,0 +1,38 @@
+package seleautomation.com.HalaTraining1;
+
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import utilities.DataReader;
+import utilities.DriverFactory;
+
+public class GitHubtest {
+
+    @DataProvider(name = "testData")
+    public Object[][] provideTestData() {
+        DataReader reader = new DataReader("\"C:\\Users\\MASS\\OneDrive\\Desktop\\QA.xlsx\""); 
```

Comment:
please use it as input like this , no need to read from excel data sheet
`
        return new Object[][] {

                { "asaltest1646449@gmail.com", "password123456789s*", false },`